frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/useCustomAPIHook.js

```diff
@@ -2,8 +2,9 @@ import { useQuery, useQueryClient } from "react-query";
 import { CustomService } from "../services/elements/CustomService";
 
 
-const useCustomAPIHook = (url, params, body, config = {}) => {
+const useCustomAPIHook = ({url, params, body, config = {},jsonPath}) => {
```

Comment:
why jsonpath @nipunarora-eGov ?

Reply:
Kept a provision to keep in name of the query